src/netbase.cpp

```diff
@@ -728,6 +724,13 @@ bool SetSocketNonBlocking(SOCKET& hSocket, bool fNonBlocking)
     return true;
 }
 
+bool SetSocketNoDelay(SOCKET& hSocket)
+{
+    int set = 1;
+    int rc = setsockopt(hSocket, IPPROTO_TCP, TCP_NODELAY, (const char*)&set, sizeof(int));
+    return rc != -1;
```

Comment:
But of course, I usually idiomatically check rc == 0 from system calls.  Not sure why I went with != -1.